### PR TITLE
Feature: Add PyPI integration

### PR DESCRIPTION
**Pull request by Bartdoekemeijer**
**Feature or improvement description**
Adds PyPI integration so that FLASC becomes pip installable without having to clone the repository from GitHub.

**Related issue, if one exists**
Issue #49.

**Impacted areas of the software**
Code deployment.

**Additional supporting information**
I have followed the style of FLORIS in the `pyproject.toml` and `python-publish.yml` files.

**Test results, if applicable**
See my test deployment on the PyPI testing environment, https://test.pypi.org/project/flasc/1.4/. I bumped this to version 1.4 because I made a mistake when uploading 1.3, and you cannot upload the code for the same version number more than once.

**Steps remaining**
- [ ] Merge this branch
- [ ] Create PyPI API token: https://pypi.org/manage/account/#api-tokens and add it to your `~/.pypirc` file.
- [ ] Publish v1.3 on PyPI using code below

```
# Building the code
python3 -m pip install --upgrade build
python3 -m build

# Uploading the packaged code
python3 -m pip install --upgrade twine
python3 -m twine upload --repository pypi dist/*  # Upload distributions to PyPI
```

### CODE REVIEW

**Comment by paulf81** [comment]
Looks good @Bartdoekemeijer I will approve and merge, thank you!

**Comment by Bartdoekemeijer** [comment]
That's great Paul, thanks!! Could you also make the PyPI API token and publish FLASC 1.3 to it? See instructions above.

**Comment by paulf81** [comment]
hi @Bartdoekemeijer , I gave it a shot today!  I made a token and followed all steps, and hit an error on the final one, have you seen this one:

```
(floris) pfleming-37929s:flasc pfleming$ python3 -m twine upload --repository pypi dist/*  # Upload distributions to PyPI
Uploading distributions to https://upload.pypi.org/legacy/
Uploading flasc-1.3-py3-none-any.whl
100% ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 51.6/51.6 kB • 00:00 • 44.4 MB/s
WARNING  Error during upload. Retry with the --verbose option for more details.
ERROR    HTTPError: 403 Forbidden from https://upload.pypi.org/legacy/
         The user 'paulf81' isn't allowed to upload to project 'flasc'. See
         https://pypi.org/help/#project-name for more information.
```

File: .github/workflows/python-publish.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
```
Comment (rafmudaf) [inline .github/workflows/python-publish.yml:26]: FYI I've added these secrets to the flasc repository

**Pull request merged**